pest/src/iterators/pair.rs

```diff
@@ -123,6 +123,35 @@ impl<'i, R: RuleType> Pair<'i, R> {
     /// ```
     #[inline]
     pub fn into_span(self) -> Span<'i> {
+        // into_span was here first, so it is left in for backwards compatibility
```

Comment:
Is there a way to mark this as deprecated?

Reply:
I'm afraid there [might be](https://github.com/rust-lang/rfcs/blob/master/text/1270-deprecation.md). Seems to be stabilized too.

Reply:
Ah! Yes you are right. I was thinking of the [stability attributes](https://github.com/rust-lang/rfcs/issues/1491).